src/buf/buf_impl.rs

```diff
@@ -127,6 +127,7 @@ pub trait Buf {
     /// This function should never panic. Once the end of the buffer is reached,
     /// i.e., `Buf::remaining` returns 0, calls to `chunk()` should return an
     /// empty slice.
+    #[cfg_attr(docsrs, doc(alias = "bytes"))]
```

Comment:
It would be nice to have a comment explaining why this alias exists.

Reply:
Done.